github/ci/services/grafana/hack/test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -ex
+
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+PROJECT_INFRA_ROOT=$(readlink --canonicalize ${BASEDIR}/../../../../../)
+
+cd $PROJECT_INFRA_ROOT
+
+cat <<EOF | kind create cluster --image quay.io/kubevirtci/kindest-node:v1.27.1 --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+EOF
+
+kubectl create ns monitoring
+
+# Install ingress
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+
+# Allow time for ingress controller to start
+sleep 20
```

Comment:
Oops, I overlooked this one, maybe we could do something similar like [there](https://github.com/kubevirt/project-infra/pull/3027/files#diff-3eb05a768dd6189427bc8a4e338855d2075e50d365080900780e770d186aa860R12) here also? 

(Could also be done in a follow up, but you decide)

Reply:
I've removed the sleep as no longer needed due to the addition of waiting on the grafana operator - but unfortunately this remove the lgtm. 

Rehearsal looks good.